### ISSUE TITLE
G5 and Android 6.0

### ISSUE THREAD

**Issue by kimberlieskandis-gpsw** [StephenBlackWasAlreadyTaken/xDrip-Experimental#357]
Does this xdrip version support G5 and Android 6?  I had been using this xdrip successfully with G5 and Android 5 but this morning I accepted the Android 6 upgrade without considering repercussions to xdrip.  After the upgrade, xdrip is no longer working.  I restarted my Samsung Galaxy Note 4 a few times.  xdrip says it is connected and authenticated to my transmitter and it shows the correct bluetooth connection.  But no data is being retrieved.  Further, it says "Cannot Calibrate right now".  The Transmitter Battery says ok, and my sensor has only been in for 8 days.  Any ideas?

**Comment by StephenBlackWasAlreadyTaken** [comment]
Yes it works with that combination but I'm not sure what version of the
beta you are on.

Android 6 requires you to allow certain permissions, head into your phone
settings, Apps, xDrip, permissions and make sure it has all the permissions
it needs (location being necessary in order to use ble on Android 6)

I hope that helps!

**Comment by kimberlieskandis-gpsw** [comment]
Thanks, that did the trick!!  Locations was turned off for xdrip.

**Comment by bellascout** [comment]
Hi Stephen, I was using Nightscout for a while and fell off the bandwagon... I am an android user and now have a G5 transmitter and receiver. Since share is only for Apple devices, do any of your apps replace this need on android phones? (Looking to connect G5 transmitter to Android phone). 
Thanks

**Comment by tzachi-dar** [comment]
Please use xDrip+.
